From 258dc6126017098007cd48988398f378bbd48deb Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Tue, 31 Oct 2023 21:47:48 +0000
Subject: [PATCH 1/1] data/metadata: rename id to fix generation with
 appstream-builder

id should match the .desktop file or a launchable desktop id should
be provided.
---
 data/metainfo/org.xfce.PanelProfiles.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/metainfo/org.xfce.PanelProfiles.appdata.xml.in b/data/metainfo/org.xfce.PanelProfiles.appdata.xml.in
index 1a9704f..157d9fa 100644
--- a/data/metainfo/org.xfce.PanelProfiles.appdata.xml.in
+++ b/data/metainfo/org.xfce.PanelProfiles.appdata.xml.in
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>xfce4-panel-profiles.desktop</id>
+  <id>org.xfce.PanelProfiles.desktop</id>
   <metadata_license>CC-BY-3.0</metadata_license>
   <project_license>GPL-3.0</project_license>
   <name>Xfce Panel Profiles</name>
-- 
2.42.0

